src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs

```diff
@@ -118,23 +118,23 @@
 
 (defn view
   []
-  (let [random-indices         (random-selection)
-        quiz-index             (reagent/atom 0)
-        incorrect-count        (reagent/atom 0)
-        show-error?            (reagent/atom false)
-        {:keys [on-success
-                on-try-again
-                masked-seed-phrase
-                back-button?
-                theme shell?]} (quo.context/use-screen-params)
-        unmasked-seed-phrase   (security/safe-unmask-data masked-seed-phrase)
-        random-phrase          (reagent/atom [])]
+  (let [random-indices  (random-selection)
+        quiz-index      (reagent/atom 0)
+        incorrect-count (reagent/atom 0)
+        show-error?     (reagent/atom false)
+        random-phrase   (reagent/atom [])]
     (fn []
       (rn/use-mount
        (fn []
          (native-module/get-random-mnemonic #(reset! random-phrase (string/split % #"\s")))))
       (when-not (empty? @random-phrase)
-        (let [current-word-index            (get random-indices
+        (let [{:keys [on-success
+                      on-try-again
+                      masked-seed-phrase
+                      back-button?
+                      theme shell?]}        (quo.context/use-screen-params)
```

Comment:
@Parveshdhull isn't conditional usage of hooks discouraged by React? Won't this generate a warning? I find very confusing to see the usage of form-2 Reagent components to manage state mixed with hooks, but I imagine you didn't want to refactor intentionally to not increase the scope of the PR?

Reply:
Thank you @ilmotta, fixed.